src/main/java/com/example/controller/SimulationController.java

```diff
@@ -221,10 +224,12 @@ public void setAvailableAlgorithms(ObservableList<AlgorithmType> algorithmTypes)
 
     public void initSimulation() {
         statisticsController.clear();
+        informationController.clearView();
```

Comment:
I'd also clear view for mode changes (simulate -> draw -> simulate).

Reply:
done